sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -590,9 +590,11 @@ def get_coder(self, coder_id):
     if coder_proto.spec.spec.urn:
       return self.context.coders.get_by_id(coder_id)
     else:
+      payload = coder_proto.spec.spec.payload
+      if isinstance(payload, bytes):
+        payload = payload.decode('utf-8')
       # No URN, assume cloud object encoding json bytes.
```

Comment:
Let's move the comment line #596 after else?

Reply:
ms